docs/reference/deploy/master-config-reference.rst

```diff
@@ -227,8 +227,16 @@ customize the UI.
 ``logo_path``
 =============
 
-Specifies the path to a user-provided logo to be shown in the UI. Ensure the path is accessible and
-reachable by the master service. The logo file should be a valid image format, with SVG recommended.
+Specifies the path to a user-provided path to the logo to be shown in the UI. Ensure the path is
```

Comment:
Should we maybe say "Specifies user provided paths to the logo to be shown in the UI. Ensure the paths are accessible and reachable by the master service."

This would probably then warrant the change of `logo_path` to `logo_paths`, but since all four paths are required now, it could be a useful change! WDYT?

Reply:
plural does make more sense although I think singular works too. I updated it.